### DISCUSSION TITLE
Option to prevent quitting app (MacOS)

### DISCUSSION THREAD

**Discussion by AlexZigante** [zoidsh/meru#267] [Feature Requests] [answered]
Option to assign Cmd+Q to have same action as Cmd-W if enabled ("hiding app"). I got so lazy by some apps working in background where Cmd+Q just hides it. If anyone else finds it useful add it as an option. Otherwise right now I use the all mighty BetterTouchTool 😅

Peace ✌️

**Comment by timche** [comment, marked as answer]
I'd suggest you to keep configuring it with BetterTouchTool. As per Apple's [Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/macos/user-interaction/keyboard/), CMD+Q is reserved to quit the app.
